Created rspatialdata blogpost (#247)
ropensci/roweb3: content/blog/2021-09-14-rspatialdata/index.md

## Retrieving demographic and health survey data using [rdhs](https://docs.ropensci.org/rdhs/)

The rdhs package (by [OJ Watson](/author/oj-watson/) and [Jeff Eaton](https://github.com/jeffeaton)) is an API client for [Demographic and Health Surveys (DHS)](https://www.dhsprogram.com/) data - which is a collection of population, health, HIV and nutrition data from more than 400 surveys in over 90 countries. This data is considered to be sensitive and hence requires one to set up an account with DHS and request permission to access the data.

Although this may look like a tedious process, the rdhs package makes it very smooth by allowing us to test out the functions of the package using model data. So one could even start on the analysis while waiting for data access rights!

Read through our tutorial on [Demographic and Health Surveys (DHS)](https://rspatialdata.github.io/dhs-data.html) for more details on how to use the rdhs package.

> Also, don't forget to have a look at our [administrative boundaries tutorial](https://rspatialdata.github.io/admin_boundaries.html) which runs you through creating outline maps of countries and their administrative divisions, our [population tutorial](https://rspatialdata.github.io/population.html) which runs you through visualizing population estimates of countries using choropleth maps and cartograms, and our tutorials on  [elevation](https://rspatialdata.github.io/elevation.html), [temperature](https://rspatialdata.github.io/temperature.html), [malaria](https://rspatialdata.github.io/malaria.html), and [air pollution](https://rspatialdata.github.io/air_pollution.html) for ideas on how to download and visualize each respective property.

Reviewer: Consider adding a heading to this section, such as "Final thoughts", or "Interested in more?" etc.
```suggestion
## Interested in more?
Have a look at our [administrative boundaries tutorial](https://rspatialdata.github.io/admin_boundaries.html) which runs you through creating outline maps of countries and their administrative divisions, our [population tutorial](https://rspatialdata.github.io/population.html) which runs you through visualizing population estimates of countries using choropleth maps and cartograms, and our tutorials on  [elevation](https://rspatialdata.github.io/elevation.html), [temperature](https://rspatialdata.github.io/temperature.html), [malaria](https://rspatialdata.github.io/malaria.html), and [air pollution](https://rspatialdata.github.io/air_pollution.html) for ideas on how to download and visualize each respective property.
```

You just read through some of the things we enjoyed while creating [rspatialdata](https://rspatialdata.github.io/). We are excited for you to try it out too! So give it a go, get creative with it, and let us know how you did! 

**Our goal for rspatialdata is not to be a comprehensive list of R packages or tutorials, but to be a starting point for anyone to find and download spatial data related to different domains, and visualize different types of spatial data objects using different approaches.**

> So if you can think of anymore R-packages that would fit into this collection or ideas on new tutorials or cool spatial visualizations, do reach out to us! Let's talk about how we can collaborate!

Reviewer: ```suggestion
So if you think of anymore R packages that would fit into this collection, ideas on new tutorials or cool spatial visualizations, do reach out to us! Let's talk about how we can collaborate!
```

Take a look at our [rainfall](https://rspatialdata.github.io/rainfall.html) and [humidity](https://rspatialdata.github.io/humidity.html) tutorials for a complete guide on how to use the nasapower package to download data and how to create cool visualizations using the data.

## Retrieving Demographic and Health Survey data using [rdhs](https://cran.r-project.org/web/packages/rdhs/index.html)

The rdhs package is an API client for the Demographic and Health Survey (DHS) Data - which is a collection of population, health, HIV and nutrition data from more than 400 surveys in over 90 countries. This data is considered to be sensitive and hence requires one to set up an account with DHS and request permission to access the data.

Although this may look like a tedious process, the rdhs package makes it very smooth by allowing us to test out the functions of the package using model data. So one could even start on the analysis while waiting for data access rights!

Read through our tutorial on [Demographic and Health Survey(DHS)](https://rspatialdata.github.io/dhs-data.html) for more details on how to use the rdhs package.

> > Our goal for rspatialdata was not to be a comprehensive list of R packages or tutorials, but to be a starting point for anyone to find and download spatial data related to different domains, and visualize different types of spatial data objects using different approaches. So give it a go and let us know how you did!

Reviewer: I think it'd be nice if you could turn this final paragraph into a bit of a conclusion, perhaps listing/linking to the other tutorials/spatial data packages you have (even if they're not rOpenSci! We want to make sure you get to promote all the hard work you've done!)
